SPECS/cloud-init-vmware-guestinfo/cloud-init-vmware-guestinfo.spec

```diff
@@ -0,0 +1,41 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+Name:           cloud-init-vmware-guestinfo
+Version:        1.3.1
+Release:        1
+Summary:        A cloud-init datasource for VMware
+Group:          System/Management
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Url:            https://github.com/vmware/cloud-init-vmware-guestinfo
```

Comment:
Consistency nit: about 80% of our current specs use "URL" in all caps for this field.